external-crates/move/crates/move-vm-types/src/values/values_impl.rs

```diff
@@ -10,6 +10,8 @@ use move_binary_format::{
     errors::*,
     file_format::{Constant, SignatureToken, VariantTag},
 };
+#[cfg(feature = "gas-profiler")]
```

Comment:
Yea, this is needed here since `ValueImpl` is private. In the future (i.e., after the most recent VM meeting) I think this will eventually all be able to live in a single crate :)